### PR TITLE
Added Race dwelf

### PR DESCRIPTION
**Pull request by deGithub6996**

### CODE REVIEW

File: homebrew/dnd-beyond/race-dwelf.xml [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+	<info>
+		<name>Dwelf</name>
+		<description></description>
+		<author url="https://www.dndbeyond.com/races/738-dwelf">Astromancer</author>
+		<update version="0.1.8">
+			<file name="race-dwelf.xml" url="https://raw.githubusercontent.com/aurorabuilder/elements/master/core/players-handbook/races/race-elf.xml" />
+		</update>
+	</info>
+
+  <element name="Dwelf" type="Race" source="D&D Beyond" id="ID_RACE_DWELF">
```
Comment (NickVendel) [inline homebrew/dnd-beyond/race-dwelf.xml]: `&` symbol is not written this way in xml, it should be `D&amp;D Beyond`

File: homebrew/dnd-beyond/race-dwelf.xml [hunk 2]
```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<elements>
+	<info>
+		<name>Dwelf</name>
+		<description></description>
+		<author url="https://www.dndbeyond.com/races/738-dwelf">Astromancer</author>
+		<update version="0.1.8">
```
Comment (NickVendel) [inline homebrew/dnd-beyond/race-dwelf.xml]: `<update version="0.1.8">` should be `<update version="0.0.1">`

**Comment by deGithub6996** [comment]
Alright, it's done. Thank you for your help.

File: homebrew/dnd-beyond/sources.xml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<elements>
+	<info>
+		<update version="0.0.1">
+			<file name="sources.xml" url="https://raw.githubusercontent.com/aurorabuilder/elements/master/homebrew/dnd-beyond/sources.xml" />
+		</update>
+	</info>
+	<element name="dnd-beyond" type="Source" source="Core" id="ID_DNDB_SOURCE_DND_BEYOND">
```
Comment (NickVendel) [inline homebrew/dnd-beyond/sources.xml]: <element name="D&amp;D Beyond"

**Pull request closed without merging**